test/provisioner/provision_ibmcloud.go

```diff
@@ -272,7 +299,7 @@ func NewIBMCloudProvisioner(properties map[string]string) (CloudProvisioner, err
 // IBMCloudProvisioner
 
 func (p *IBMCloudProvisioner) CreateCluster(ctx context.Context, cfg *envconf.Config) error {
-	ibmcloudTrace("CreateCluster()")
+	log.Info("CreateCluster()")
```

Comment:
I think this could have the `Debug` level?

Reply:
Agree, thanks @stevenhorsman